src/NosCore.Parser/Parsers/ItemParser.cs

```diff
@@ -0,0 +1,643 @@
+                {nameof(ItemDto.Height), chunk => ImportHeight(chunk)},
+                {nameof(ItemDto.DefenceDodge), chunk => ImportDefenceDodge(chunk)},
+                {nameof(ItemDto.CloseDefence), chunk => ImportCloseDefence(chunk)},
+                {nameof(ItemDto.DistanceDefence), chunk => ImportDistanceDefence(chunk)},
+                {nameof(ItemDto.MagicDefence), chunk => ImportMagicDefence(chunk)},
+                {nameof(ItemDto.BasicUpgrade), chunk => ImportBasicUpgrade(chunk)},
+                {nameof(ItemDto.WaitDelay), chunk => ImportWaitDelay(chunk)},
+                {nameof(ItemDto.ElementRate), chunk => ImportElementRate(chunk)},
+                {nameof(ItemDto.Speed), chunk => ImportSpeed(chunk)},
+                {nameof(ItemDto.SpType), chunk => ImportSpType(chunk)},
+                {nameof(ItemDto.LevelJobMinimum), chunk => ImportLevelJobMinimum(chunk)},
+                {nameof(ItemDto.ReputationMinimum), chunk => ImportReputationMinimum(chunk)},
+                {nameof(ItemDto.ItemValidTime), chunk => ImportItemValidTime(chunk)},
+                {nameof(ItemDto.Element), chunk => ImportElement(chunk)},
+                {nameof(ItemDto.MaxCellonLvl), chunk => ImportMaxCellonLvl(chunk)},
+                {nameof(ItemDto.MaxCellon), chunk => ImportMaxCellon(chunk)},
+            };
+
+            var genericParser = new GenericParser<ItemDto>(folder + ItemCardDto,
+                "#========================================================", 1, actionList, _logger);
+            var items = genericParser.GetDtos().GroupBy(p => p.VNum).Select(g => g.First()).ToList();
+            foreach (var item in items)
+            {
+                HardcodeItem(item);
+                if (item.ItemType == ItemType.Specialist)
+                {
+                    var elementdic = new Dictionary<ElementType, int> {
+                        { ElementType.Neutral, 0 },
+                        { ElementType.Fire, item.FireResistance },
+                        { ElementType.Water, item.WaterResistance },
+                        { ElementType.Light, item.LightResistance },
+                        { ElementType.Dark, item.DarkResistance }
+                    }.OrderByDescending(s => s.Value);
+
+                    item.Element = elementdic.First().Key;
+                    if (elementdic.First().Value != 0 && elementdic.First().Value == elementdic.ElementAt(1).Value)
+                    {
+                        item.SecondaryElement = elementdic.ElementAt(1).Key;
+                    }
+                }
+            }
+            SetVehicles(items, new Dictionary<byte, List<(short, short)>>
+            {
+                { 20, new List<(short, short)> { (9054, 2368), (1906, 2368), (9055, 2370), (1907, 2370), (9058, 2406), (1965, 2406), (9065, 2411), (5008, 2411), (5238, 1817), (5226, 1817), (5240, 1819), (5228, 1819), (5234, 2522), (5236, 2524) } },
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 248 characters long line (which is greater than 200 authorized).](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4456396)